airflow/models/taskinstance.py

```diff
@@ -617,6 +619,11 @@ def prev_attempted_tries(self) -> int:
     def next_try_number(self) -> int:
         return self._try_number + 1
 
+    @property
+    def operator_name(self) -> str:
+        """@property: use a more friendly display name for the operator, if set."""
+        return self.custom_operator_name or self.operator  # type: ignore
```

Comment:
Let’s add a `or ""` at the end just to be safe. Leaking `None` here can be pretty problematic.